core/modules/language/language.admin.inc

```diff
@@ -78,7 +78,7 @@ function language_admin_overview_form($form, &$form_state) {
   $form['actions'] = array('#type' => 'actions');
   $form['actions']['submit'] = array(
     '#type' => 'submit',
-    '#value' => t('Save configuration'),
+    '#value' => t('Save order'),
```

Comment:
We also set enabled/disabled and default language via that form, so "Save configuration" seems to be appropriate.

Reply:
Ah I see. The tests will also need to be un-updated.